app.json

```diff
@@ -80,7 +80,7 @@
     "S3_ACCESS_KEY":{
       "required":true
     },
-    "S3_ACCESS_KEY_ID":{
+    "SHOW_SALE_LISTINGS": {
```

Comment:
@akegan was there a reason to remove the `S3_ACCESS_KEY_ID` env var instead of placing the `SHOW_SALE_LISTINGS` env var below it?

Reply:
fixed!